Library/Homebrew/cmd/tests.rb

```diff
@@ -1,13 +1,22 @@
 #: @hide_from_man_page
-#:  * `tests` [`-v`] [`--coverage`] [`--generic`] [`--no-compat`] [`--only=`<test_script/test_method>] [`--seed` <seed>] [`--trace`] [`--online`]:
+#:  * `tests` [`-v`] [`--coverage`] [`--generic`] [`--no-compat`] [`--only=`<test_script/test_method>] [`--seed` <seed>] [`--trace`] [`--online`] [`--official-cmd-taps`]:
 #:    Run Homebrew's unit and integration tests.
 
 require "fileutils"
 
 module Homebrew
   def tests
+    ENV["HOMEBREW_NO_ANALYTICS_THIS_RUN"] = "1"
+
+    if ARGV.include? "--official-cmd-taps"
+      ENV["HOMEBREW_TEST_OFFICIAL_CMD_TAPS"] = "1"
+      OFFICIAL_CMD_TAPS.each do |tap, _|
+        tap = Tap.fetch tap
```

Comment:
This probably needs a `require "tap"` at the top of the file. Here's what I currently get:

```console
$ brew tests --no-compat --official-cmd-taps
Error: uninitialized constant Homebrew::Tap
Please report this bug:
    https://git.io/brew-troubleshooting
/opt/brewery/dummy/Library/Homebrew/cmd/tests.rb:14:in `block in tests'
/opt/brewery/dummy/Library/Homebrew/cmd/tests.rb:13:in `each'
/opt/brewery/dummy/Library/Homebrew/cmd/tests.rb:13:in `tests'
/opt/brewery/dummy/Library/brew.rb:87:in `<main>'
```